docs/cis_self_assessment.md

```diff
@@ -2385,6 +2128,7 @@ TLS ciphers have had a number of known vulnerabilities and weaknesses, which can
 **Result:** Pass
```

Comment:
I think its a stretch to as we pass this, since we do not actually pass the audit. I'm going to open an issue to set the specified ciphers when in cis mode. Can you mark this as fail todo until then?

Reply:
Actually, lets make this "Not Scored - Operator Dependent"

I'll update the remediation.

Reply:
Can you add the audit from the upstream doc

Reply:
Let's make this "Not Scored - Operator Dependent"